src/NautilusRefactoring/NautilusRefactoring.class.st

```diff
@@ -154,7 +154,10 @@ NautilusRefactoring >> shouldExtractAssignmentTo: aString [
 
 { #category : #option }
 NautilusRefactoring >> shouldInlineExpression: aString [
-	^ self confirm: ('Do you want to inline the expression ''<1s>'' in the current method?' expandMacrosWith: aString)
+
+	^ (self confirm:
+		   ('Do you want to extract the expression ''<1s>'' into a variable in the current method?' 
```

Comment:
Why are you not using the expandMacros with both parts, the expression and the method selector?